index.html

```diff
@@ -105,7 +105,7 @@ <h2 class="name">
 
     <div class="container-works-section" id="portfolio">
       <!-- the first work -->
```

Comment:
- [ ] Good job generating your project section dynamically from JavaScript. Kindly remove all the commented code from the index.html file so as to keep your work clean 👍